pkg/controller/migmigration/quiesce.go

```diff
@@ -20,6 +24,7 @@ func (t *Task) quiesceApplications() error {
 	err = t.suspendCronJobs(client)
 	if err != nil {
 		log.Trace(err)
+		return err
```

Comment:
Was the change in behavior intentional here? We now don't even try to quiesce any other resource type once one type gives us an error. Previously if CronJob suspend failed we'd still try to scale down Deployments, etc. I'm not sure which approach is better overall, but if there is some fundamental problem with scaling down CronJobs in a particular cluster, we'll now never scale down pods at all. That said, it looks like the old error handling was somewhat broken, since if one type returned an error, that error got swallowed by the next scaleDown call. If we want to preserve prior behavior, we'll need to build up an errors slice.

Reply:
Actually, nevermind that. We probably do want to fail on one error. If even one of these fails, then the ensure step will fail forever anyway.